streams/src/test/java/org/apache/kafka/streams/kstream/internals/SuppressTopologyTest.java

```diff
@@ -62,16 +62,16 @@
         "      --> myname\n" +
         "      <-- KSTREAM-SOURCE-0000000005\n" +
         "    Processor: myname (stores: [myname-store])\n" +
-        "      --> KTABLE-TOSTREAM-0000000006\n" +
+        "      --> KTABLE-TOSTREAM-0000000008\n" +
```

Comment:
Same here, since we'll leave `suppress` as is, these test changes can be reverted.

\cc @vvcephei

Reply:
Yep, we should not change these tests. Thanks!